 active-rfcs/0001-issue-helper.md

```diff
@@ -0,0 +1,106 @@
+- Feature Name: Issue Helper
+- Start Date: 2020-02-20
+- Reference Issues:
+- Implementation PR:
+
+# Summary
+
+We'll use GitHub Issue [contact_links](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) to create issues, and in contact_links we'll put an external web page called issue-helper to help us create issues in a standard format.
+
+# Motivation
+
+In the past, TiDB's code repository used GitHub Issues to create bug reports, but because GitHub Issues use markdown to organize and write the report content. This caused our bug reports to be missing the basic information needed to reproduce the bug, making it impossible for our developers to reproduce the bug.
+
+Also, as our bug reports require more and more information, the issue templates become more and more complex, making it difficult for users or contributors to fill out the templates.
+
+So we can use GitHub Issue's contact_links to develop a web page to help and guide people through the process of filling out a bug report. This will help us collect bug reports in a better and more standard way and make it easier for users and contributors to fill them out.
+
+# Detailed design
+
+## Introducing GitHub Issue contact_links
+
+GitHub Issues provides a way to configure external links. We can use the following configuration to add an external contact link.
+
+```yaml
+// .github/ISSUE_TEMPLATE/config.yml
+blank_issues_enabled: false
+contact_links:
+  - name: Ask TiDB User Group Support
+    url: https://asktug.com/
```

Comment:
AskTug is now only for China, is it proper here?

Reply:
Maybe we specify in the description that only Chinese is supported?

Reply:
Maybe we can remove this option. When users want to create a Github issue, they may not need Asktug; in addition, we can strengthen the capability of the issue helper step by step, it doesn't require users to think about choosing the problem-solving entry anymore.

Reply:
I agree.

Reply:
Removed.